gettext doesn't convert charsets

**Describe the bug**
I have an application which keeps translations in UTF-8, to display them in a user's native encoding I use `bind_textdomain_codeset(const char * domainname, const char * codeset )` but calls to `gettext()` return strings in UTF-8.

**Environment**
- OS: Windows
- Compiler: MSVC2015u3

**Expected behavior**
Strings returned by `gettext()` are converted to local encoding.

@375gnu I think gettext doesn't really use iconv right now.
Because only `DEPENDS_ON_LIBICONV` is defined in CMakeLists.txt and `HAVE_ICONV` is not defined, but in the code:
```
#if DEPENDS_ON_LIBICONV && HAVE_ICONV
```

Rebuilt after adding -DHAVE_ICONV, an error occurred:
```
gettext-runtime\intl\dcigettext.c(1316): error C2065: 'ICONV_CONST': undeclared identifier
```
`ICONV_CONST` definition not found after looking up the header file of libiconv.

--- a/ports/gettext/CMakeLists.txt
+++ b/ports/gettext/CMakeLists.txt
@@ -102,7 +102,7 @@ if(BUILD_SHARED_LIBS)
 endif()
 add_definitions("-DBUILDING_LIBINTL -DIN_LIBINTL -DENABLE_RELOCATABLE=1 -DIN_LIBRARY")
 
-add_definitions("-DNO_XMALLOC -Dset_relocation_prefix=libintl_set_relocation_prefix -Drelocate=libintl_relocate -DDEPENDS_ON_LIBICONV=1 -DHAVE_CONFIG_H -D_CRT_SECURE_NO_WARNINGS")
+add_definitions("-DNO_XMALLOC -Dset_relocation_prefix=libintl_set_relocation_prefix -Drelocate=libintl_relocate -DDEPENDS_ON_LIBICONV=1 -DHAVE_CONFIG_H -DHAVE_ICONV -DHAVE_ICONV_H -DICONV_CONST=const -D_CRT_SECURE_NO_WARNINGS")
 
 add_library(libintl ${SOURCES})
 target_link_libraries(libintl PRIVATE unofficial::iconv::libcharset unofficial::iconv::libiconv)

--- a/ports/gettext/CONTROL
+++ b/ports/gettext/CONTROL
@@ -1,5 +1,5 @@
 Source: gettext
-Version: 0.19-11
+Version: 0.19-12
 Homepage: https://www.gnu.org/software/gettext/
 Description: The GNU gettext utilities are a set of tools that provides a framework to help other GNU packages produce multi-lingual messages. Provides libintl.
 Build-Depends: libiconv

